build.gradle

```
@@ -305,6 +306,18 @@ subprojects {
       }
     }
 
+    ideProbe = { ->
+      repositories {
+        maven { url = "https://jetbrains.bintray.com/intellij-third-party-dependencies" }
```

Nit: for consistency
```suggestion
        maven { url 'https://jetbrains.bintray.com/intellij-third-party-dependencies' }
```